tests/TestCase/View/Helper/BakeHelperTest.php

```diff
@@ -119,4 +119,105 @@ public function testAliasExtractorBelongsToMany()
         $expected = ['tags'];
         $this->assertSame($expected, $result);
     }
+    
+    /**
+     * test stringifyList empty
+     *
+     * @return void
+     */
+    public function testStringifyListEmpty()
+    {
+        $result = $this->BakeHelper->stringifyList([]);
+        $expected = '';
+        $this->assertSame($expected, $result);
+    }
+    
+    /**
+     * test stringifyList defaults
+     *
+     * @return void
+     */
+    public function testStringifyListDefaults()
+    {
+        $list = ['one' => 'foo', 'two' => 'bar', 'three'];
+        $result = $this->BakeHelper->stringifyList($list);
+        $spaces = '    ';
+        $expected = "\n".
+            $spaces.$spaces."'one' => 'foo',\n".
+            $spaces.$spaces."'two' => 'bar'\n".
+            $spaces.$spaces."'three'\n".
```

Comment:
Spaces around operators improve readability and consistency with the rest of the project.